manifests/charts/ambient/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Copied from `helm-profiles/ambient.yaml`
```

Comment:
can you not just do literally `profile: ambient`?

Reply:
You cannot, but lemme make sure I'm not missing some non-obvious syntax.

Reply:
s/cannot/can/g - TY!